specification/glossary.md

```diff
@@ -46,7 +46,7 @@ A customer's agreement to consume a specific quantity of a service or resource o
 
 <a name="glossary:commitment-based-discount"><b>Commitment-Based Discount</b></a>
 
-Also known as Commitment Discount, this is a commitment for an amount of usage or spend throughout a specified term, in exchange for discounted unit pricing on that amount. The commitment may be based on quantities of resource units or monetary value, with various payment options and time frames.
+A billing discount model that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term.  Commitment-based discount purchases, made upfront and/or with recurring monthly payments are amortized evenly across predefined charge periods (i.e. hourly), and unused amounts cannot be carried over to subsequent charge periods. Commitment discounts are publicly available to customers without special contract arrangements.
```

Comment:
have we defined amortization in the glossary? Effectivecost is the amortization amount but doesnt speak to amortization. If we haven't defined amortization in the glossary then would it be better to say 'payments are distributed evenly across'

Reply:
Matches this definition exactly, so leaving as is: https://www.finops.org/assets/terminology/#commitment-discounts